src/libs/actions/IOU.ts

```diff
@@ -1061,12 +1062,18 @@ function setMoneyRequestDistanceRate(transactionID: string, customUnitRateID: st
     }
 
     const distanceRate = DistanceRequestUtils.getRateByCustomUnitRateID({policy, customUnitRateID});
+    const transaction = isDraft ? allTransactionDrafts[`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transactionID}`] : allTransactions[`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`];
+    let newDistance;
+    if (distanceRate?.unit && distanceRate?.unit !== transaction?.comment?.customUnit?.distanceUnit) {
+        newDistance = DistanceRequestUtils.convertDistanceUnit(getDistanceInMeters(transaction, transaction?.comment?.customUnit?.distanceUnit), distanceRate.unit);
```

Comment:
I think we should import these functions individually instead of using `DistanceRequestUtils.[...]` - I think lint probably should have told you to do this? Not sure why it didn't...

Reply:
Yeah totally, it's definitely not a change you made, but we should follow this new rule - https://expensify.slack.com/archives/C01GTK53T8Q/p1736838528732529 - where possible. Again, I'm not sure why lint didn't call out the rule, but let's proactively update it here.

Reply:
ah okay i guess we can leave it for now